src/org/omegat/core/data/PluginInformation.java

```diff
@@ -121,6 +187,34 @@ public String getAuthor() {
         return author;
     }
 
+    public final String getCategory() {
+        return category;
+    }
+
+    public final String getLink() {
+     return link;
+    }
+
+    public final File getJarFile() {
+        return new File(url.getPath().substring(5, url.getPath().indexOf("!")));
```

Comment:
This string munging makes me very nervous. What is the magic `5`? Is there really no better way to extract the file path?

Reply:
This can be safely delete

5 = 'file:`.length

Reply:
I've replaced with getUrl() here.